app/front_page/content.html

```diff
@@ -3,23 +3,23 @@
 
   <div class="col-xs-3">
   <select class="form-control" ng-model="vm.transfer.type" ng-disabled="vm.transfer.components.length > 0">
-    <option value="standard" selected>Standard</option>
-    <option value="unzipped bag">Unzipped Bag</option>
-    <option value="zipped bag">Zipped bag</option>
-    <option value="dspace">DSpace</option>
-    <option value="disk image">Disk Image</option>
+    <option value="standard" selected>{{ "Standard" | translate }}</option>
```

Comment:
Why ` <option value="standard">{{ "Standard" | translate }}</option>` instead of ` <option value="standard" translate>Standard</option>` I think they're equivalent.